Overview.bs

```diff
@@ -428,8 +429,14 @@ opt to exclude those unused features from the subset definition.
 The following algorithm is used by a client to extend an [=incremental font|incremental=] [=font subset=] to cover additional
 code points, layout features and/or design space. This algorithm incrementally selects and applies patches to the font one at a time, loading
 them as needed. As an important optimization to minimize network round trips it permits loads to be started for all patches that will eventually
-be needed. [[#font-patch-invalidations]] is used to determine which patches loads can be started for. Any patches which match the target subset definition
-and will not be invalidated by the next patch to be applied according to [[#font-patch-invalidations]] can be expected to be needed in future iterations.
+be needed. These come in two forms:
+
+*  First, [=patch map|patch mapping entries=] may list more than one URI, where each URI after the first is to be preloaded because it
```

Comment:
Aren't we using "preloading" for having the "initial" file already include some patches?

Reply:
Ok, reworked this to avoid "preload"